tests/lib/nm/ipv6_test.py

```diff
@@ -117,3 +123,27 @@ def test_get_info_with_ipv6_config():
             }
         ]
     }
+
+
+def test_create_setting_with_link_local_addresses(NM_mock):
+    config = {
+        'enabled': True,
+        'address': [
+            {'ip': 'fe80::1', 'prefix-length': 64},
```

Comment:
I would use constants so the reader will know which one is the link-local and which one is not, but it is not a must. Your call.